Gemfile.lock

```diff
@@ -299,6 +302,7 @@ DEPENDENCIES
   jsbundling-rails
   open-uri
   pg (~> 1.1)
+  pg_search
```

Comment:
Looks like this is in the gem file twice - once on 178 and once on 305. I don't think it'll break anything, but something to fix in the future!